.coffeelintignore

```diff
@@ -1,2 +0,0 @@
-iconsets
-node_modules
```

Comment:
eslint has an equivalent `.eslintignore`. but since eslint ignores `node_modules` by default and there are no `.js` files in `iconsets`, I felt it unnecessary to add.